queries.sql

```diff
@@ -188,4 +188,19 @@ SELECT vets.name, species.name, count(species.name) from vets
   JOIN species on animals.species_id = species.id 
   WHERE vets.name = 'Maisy SMith' 
   GROUP BY species.name, vets.name 
-  ORDER BY count DESC lIMIT 1;
\ No newline at end of file
+  ORDER BY count DESC lIMIT 1;
+
+  /* Perfomence audit */
+  CREATE INDEX idx_animal_id ON visits (animal_id);
+
+SET work_mem = '128MB';
+
+ANALYZE visits;
+
+CREATE INDEX idx_email ON owners (email);
+
+ANALYZE owners;
+
+CREATE INDEX idx_vet ON visits (vet_id);
```

Comment:
- [ ] all the CREATE queries should be put in the `schema.sql` file

Reply:
ok